addons/mail/static/lib/odoo_sfu/odoo_sfu.js

```diff
@@ -15824,6 +15824,11 @@ class SfuClient extends EventTarget {
         camera: null,
         screen: null,
     };
+    /** @type {Object<"audio" | "video", import("mediasoup-client").types.ProducerOptions>} */
+    _producerOptionsByKind = {
+        audio: DEFAULT_PRODUCER_OPTIONS,
+        video: DEFAULT_PRODUCER_OPTIONS,
+    };
```

Comment:
If the default value for video and audio are the same, why do we need this split?

Reply:
The server sends an object that has different values for each entries, this default value is in case the server does not provide producerOptions for backwards compatibility.